app/cli.py

```diff
@@ -209,6 +210,11 @@ def _update_datasets(app):
                 print("\033[0m")
                 continue
 
+        # The folowing relates to the DATS.json files of the projects. Skip directories that aren't projects or investigators.
+        patterns = [app.config['DATA_PATH'] + '/conp-dataset/projects/*', app.config['DATA_PATH'] + '/conp-dataset/investigators/*']
```

Comment:
you can remove investigator. It doesn't exists anymore. 
If we ever add it back we will change this line.